src/model/primary_key_index.py

```diff
@@ -0,0 +1,16 @@
+from model.index import *
+
+
+class PrimaryKeyIndex(Index):
```

Comment:
You should implement method compare here. It should return MIN_INTEGRT which mean it is the most efficient index